aresponses/main.py

```diff
@@ -90,7 +96,33 @@ def _host_matches(self, match_host):
                    (match_querystring and _text_matches_pattern(path_pattern, path_qs)):
                     path_matched = True
                     if _text_matches_pattern(method_pattern, method.lower()):
+                        if response is PassThrough:
+                            connector = TCPConnector()
+                            connector._resolve_host = partial(self._old_resolver_mock, connector)
```

Comment:
Is this unmocking the class or just this instance? Would be good to have a test for that.

Reply:
only this instance (if you mean the connector)

Reply:
if you mean the `ClientRequest`, it is unmocking whole class - this may cause problems if the passthrough and mocked requests are made concurrently.